neurotech/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="/assets/external/bootstrap.min.css">
+        <link rel="stylesheet" href="/assets/external/bootstrap-icons.css">
+        <link rel="stylesheet" href="/assets/ieee.css">
+        <link rel="stylesheet" href="./neurotech.css">
+        <link rel="icon" href="/favicon.ico"/>
+
+        <meta name="description" content="The official website of the Purdue IEEE Student Branch">
+        <meta name="author" content="Purdue IEEE">
+
+        <title>Purdue IEEE</title>
+
+        <script src="/assets/external/bootstrap.bundle.min.js"></script>
+        <script src="/assets/external/vue2.min.js"></script>
+        <script src="/assets/include.js"></script>
+
+    </head>
+    <body>
+    <div id="app" class="bg-dark">
+        <ieee-header head="Purdue IEEE" page="Neurotech"></ieee-header>
+
+        <main>
+            <div class="container-fluid pt-5 mt-5 text-center bg-dark">
```

Comment:
This section doesn't follow the existing format of the other tech committee pages.

There should be the logo + title in the top bar with nav links, then a paragraph of text describing the committee with a catchy marketing title.

Reply:
Yeah I think it is fine for now it is similar to what we did to software saturdays